src/state/slices/common-selectors.ts

```diff
@@ -57,10 +57,16 @@ export const selectPortfolioAssetBalancesBaseUnit = createDeepEqualOutputSelecto
   selectPortfolioAccountBalancesBaseUnit,
   (accountBalancesById): Record<AssetId, string> =>
     Object.values(accountBalancesById).reduce<Record<AssetId, string>>((acc, byAssetId) => {
-      Object.entries(byAssetId).forEach(
-        ([assetId, balance]) =>
-          (acc[assetId] = bnOrZero(acc[assetId]).plus(bnOrZero(balance)).toFixed()),
-      )
+      Object.entries(byAssetId).forEach(([assetId, balance]) => {
+        const bnBalance = bnOrZero(balance)
+
+        // don't include assets with zero crypto balance
+        if (bnBalance.isZero()) {
+          return
+        }
```

Comment:
Fix here. Prevents the app thinking you hold e.g `0.000000000000000000000000000000000000000000 PEPE` after selling max.

Reply:
This wasnt obvious until we listed the portfolio assets by balance in the the asset search.

Reply:
Very nice, looks like this may fix some balances in dashboard being displayed despite being zero-value when you scroll down on the infinite scroll too?